pkg/apis/core/validation/validation_test.go

```diff
@@ -3713,7 +3713,7 @@ func TestValidateEnvFrom(t *testing.T) {
 			envs: []core.EnvFromSource{
 				{},
 			},
-			expectedError: "field: Invalid value: \"\": must specify one of: `configMapRef` or `secretRef`",
+			expectedError: "field: configMapRef,secretRef: Required one of values: \"\": must specify at least one",
```

Comment:
this reads strange, please see suggestions below.

Reply:
Perhaps it should read something like: "field**s**: x,y,z: exactly one must have a value."